js/components/AudioPlayer.react.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * AudioPlayer
+ */
+
+import { asyncPlayAudio, asyncPauseAudio } from '../actions/AppActions';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+class AudioPlayer extends Component {
+	render() {
+		const { dispatch, src } = this.props;
```

Comment:
@calebwoods My coworkers' bad tabbing manners have rubbed off on me... I converted this file to 2-space tabbing.